config/actions/gradle-assemble-and-bundle/action.yml

```diff
@@ -2,15 +2,15 @@ name: 'Run gradle assemble and bundle (release build type)'
 description: 'Run the "assembleRelease" and "bundleRelease" commands for a Gradle project'
 
 inputs:
-  access-token:
+  github-token:
```

Comment:
We can get rid of these right - they'll be set at the workflow level so don't need to ingest them as inputs.

Reply:
Ignore me, these apply to a custom action not a workflow_dispatch